README.md

```diff
@@ -66,7 +60,7 @@ Please note that using Ollama requires a large (>4GB) download of the docker ima
     ```
 
 
-    This will start Ollama and a PostgreSQL instance with the pgai extension installed. 
+    This will start Ollama and a PostgreSQL instance. 
```

Comment:
I think we want to highlight that if the user already has a postgresdb (and potentially a model API) they can use those as-is further up. It's a bit odd for something branded as a library to require a compose I think.